test/unit/defaults_test.rb

```diff
@@ -0,0 +1,32 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+describe HammerCLIForeman::Defaults do
+
+  context "Defaults" do
+
+    defaults_provider = HammerCLIForeman::Defaults.new
+    user = {"results" => ["default_organization" => {"id" => 1}, "default_location" => {"id" => 1}]}
```

Comment:
could you use different id values to avoid possibility overseeing of swapping them by mistake?